src/NetAppFiles/NetAppFiles/help/Restore-AzNetAppFilesSnapshotFile.md

```diff
@@ -0,0 +1,260 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.NetAppFiles.dll-Help.xml
+Module Name: Az.NetAppFiles
+online version:
+schema: 2.0.0
+---
+
+# Restore-AzNetAppFilesSnapshotFile
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```

Comment:
Please fill placeholder

Reply:
hmm I did, must have gotten lost after regeneration of docs, will add.

Reply:
btw there are a lot of RELATED LINKS that could be added, do we have a guidance for this section?

Reply:
No guidance. they are nice to have.